keps/sig-architecture/2019-07-16-node-role-label-use.md

```diff
@@ -175,22 +147,54 @@ Some components like the external cloud provider controllers (considered to fall
 
 ## Design Details
 
+### Migrating existing deployments
```

Comment:
is this just a move or did stuff change (other than the addition of release 5)?

Reply:
Moved and slightly cleaned up wording.